1.0.1 changes

cs-abhishek-shukla opened the pull request:
- Edited the following for typos, grammar, missing information, consistency, and clarity
  - release_notes.md
  - README.md
  - setup.md
  - usage.md
  - contents.md

cs-rohit-khune commented on release_notes.md:
```diff
@@ -1,6 +1,8 @@
+
+- Removed the **Scenario** module and its **System View Template (SVT)** from the solution pack since it is already a part of the SOC Simulator solution pack
+- The example alert generating playbook **Generate Phishing Email Alert** now creates alerts of type **Suspicious Email**
+- The response playbook **Phishing Email Triage Based On ML Classifier** runs when alerts of type **Suspicious Email** are created
```
@cs-abhishek-shukla This point needs to be changed. it is a Manually triggered playbook. Previously it is configured to execute for a 'Phishing' type alert. Now it is updated to execute only for 'Suspicious Email' type Alert

cs-abhishek-shukla committed:
1.0.1 changes
- Edited following for typos, grammar, missing information, consistency, and clarity
  - release_notes.md
  - README.md
  - setup.md
  - usage.md
  - contents.md
